### ISSUE TITLE
Live Tv Issues here

### ISSUE THREAD

**Issue by Tempest0580** [Tempest0580/Temptv#34]
We don't need all these tickets open for the same issue, Put all your issues here.
Thanks

**Comment by Tempest0580** [comment]
I'm Working on Tying to fix UstvGo, I'm going to push a update that will fix Some of the channels.
But the ones that are VPN locked will take more time to try and fix.
Thanks

**Comment by bjgood** [comment]
I think arconaitv is messed up. Not working on my end, at least.

Great work, Tempest.

**Comment by Tempest0580** [comment]
Arconaitv has changed code on the site to stop kodi use. I'm working on trying to find a fix or a replacement. 
Thanks

**Comment by bjgood** [comment]
USTVGo is working on my end. What channels are down? I don't watch all of them, of course.

**Comment by Tempest0580** [comment]
HBO and showtime I think are just 2 that require a VPN now. I have heard some people in Canada dont need a vpn for any.

I'll be pushing another update later with a different site. 99% of the channels work. It even has some channels that ustvgo doesnt have.

Some of the channels in ustvgo and zynet.tv should be working again in the next update too.
Thanks

**Comment by Tempest0580** [comment]
Pushed a new update. Added a new source.  
Let me know how it works.
Thanks

**Comment by Dragon990** [comment]
It's not working

**Comment by TGMisfit** [comment]
I just wanted to say I can watch ZyneTV fine, but not USTVGo, but it looks like that's being worked on. Thanks for the great service!

**Comment by Tempest0580** [comment]
UstvGo has changed it code again. I'm not going to rush to try and fix it again. I'm spending more time trying to fix it every 5 mins then trying to find/code other sources.
Try Imkmedia. It seems to be decent on my end.
Thanks

**Comment by TGMisfit** [comment]
OK.

**Comment by derrgoo** [comment]
Thanks for all your work. I know I watch mostly cnn. Vpn or no vpn is working.  But the website is okay for now.

**Comment by derrgoo** [comment]
> I just wanted to say I can watch ZyneTV fine, but not USTVGo, but it looks like that's being worked on. Thanks for the great service!

Which channels work for you? I tried vpn also don’t work.

**Comment by derrgoo** [comment]
> UstvGo has changed it code again. I'm not going to rush to try and fix it again. I'm spending more time trying to fix it every 5 mins then trying to find/code other sources.
> Try Imkmedia. It seems to be decent on my end.
> Thanks

Agreed. With everyone at home they must be trying to keep us out with the high traffic. :-)

**Comment by jabales** [comment]
Are all the sources(Streamlive, Zyne etc.) down for anyone else? I know Tempest0580 is working on USTVGo, but none of the others are working for me the last couple days. Nothing can even pull an M3U link or channel data.

**Comment by YKL66** [comment]
[New Source]: I found a source or webpage for you to add. Not sure where to post this, but didn't want to make another tread. This is the web address: (www.stream2watch.ws) Hope this helps for future addons.

**Comment by Tempest0580** [comment]
I have looked into the site a few times. It re streams from ustvgo and a few other sites. I'm looking in to scraping a couple of the sites now. But most of them have shitty sources. The streams go down constantly.  But I'm working on trying to scrape the best once. Mainly as backups. 
Thanks

**Comment by Tempest0580** [comment]
Added a radio station in the new update. Still working on ustvgo. Hopefully it will be ready soon.
Thanks

**Comment by derrgoo** [comment]
> Added a radio station in the new update. Still working on ustvgo. Hopefully it will be ready soon.
> Thanks

Sounds great!

**Comment by YKL66** [comment]
@Tempest0580:  Thank you again for these awesome add-ons, the Imkmedia is a great add. I was gonna ask you to add a Canadian channel ctv sci-fi and saw that the space channel is the same. The mashup of all my scifi 24-7. Thank you for the hard work that you do! 👍

**Comment by jabales** [comment]
@YKL66  Yeah, Imkmedia is really nice. Haven't seen a free iptv service that offers as many HD channels as it does. Even the standard definition channels are much higher resolution then USTVGo.

Again, thank you for your support, @Tempest0580 .

**Comment by Tempest0580** [comment]
I agree. I hope the site just gets better.
Thanks

**Comment by Tempest0580** [comment]
So I crack arconaitv channels a few days ago. Been testing it. Been debating on if I should add them back to the addons. Looking for input. 
Thanks

**Comment by JayDee696969** [comment]
> So I crack arconaitv channels a few days ago. Been testing it. Been debating on if I should add them back to the addons. Looking for input.
> Thanks

It would be nice especially since that was the last source left for 24/7 channels, since Sportdevil stopped working, and most addons pulled the 24/7 streams from Sportsdevil which i have a feeling pulled from Acronai, lol.

**Comment by derrgoo** [comment]
> So I crack arconaitv channels a few days ago. Been testing it. Been debating on if I should add them back to the addons. Looking for input.
> Thanks

Humm. Hard decision I bet. Well done as usual.

**Comment by derrgoo** [comment]
> So I crack arconaitv channels a few days ago. Been testing it. Been debating on if I should add them back to the addons. Looking for input.
> Thanks

By the way I see that crew added it. Fyi

**Comment by YKL66** [comment]
@derrgoo - Yeah I'm using them also, didn't know if I could post that here to help with the addon.

**Comment by Tempest0580** [comment]
I'll be adding a fix for arconaitv in tempest, temptv and arconaitv addon today. 
Thanks

**Comment by Tempest0580** [comment]
If everyone can go to https://www.arconaitv.us/ and donate something. Maybe we can get enough donations in so he wont change his code and and stop us from using it.
Thanks

**Comment by jabales** [comment]
If these cat and mouse games keep up for long enough aren't either of your codes going to be changed so much that they don't work, eventually?

**Comment by Tempest0580** [comment]
> If these cat and mouse games keep up for long enough aren't either of your codes going to be changed so much that they don't work, eventually?

Yes but I'm hoping if we get enough people that can donate and show him it's worth keeping the kodi users able to use his stream he wont change the code to where it wont work. It's a long shot because I'm the only dev that is trying to get people to donate but I think its worth a shot. I think the streams are worth a small donation from everyone if they can. 
Thanks

**Comment by derrgoo** [comment]
Well done 🙏

**Comment by Tempest0580** [comment]
Added UstvGo streams back, Needs testing and some channels will not work without a vpn.
Thanks

**Comment by Tempest0580** [comment]
I'm should be pushing a update tonight to fix ustvgo and a couple others. Thanks

**Comment by cowfish64** [comment]
hi 1st I want to thank you for all the hard work.

I am using a vpn and Ustvgo just a couple of days go, for two day i cannot connect.  
Also CNN is missing from the menu in both tempest and temptv.
Just wondering if I am the only one.
Thanks

**Comment by Tempest0580** [comment]
I'm going to look into it this weekend but it seems ustvgo has changed its code once again. Thanks

**Comment by suzie59** [comment]
Mine is still not working

**Comment by israelchoen** [comment]
UsaTv Go works great.
When I try to log in to Myustv and YourSports I get an error message. I also can't log in to Ustreamix.
Sports24 log in but most channels do not work.

**Comment by suzie59** [comment]
the channels I would like to watch are blocked saying I need vpn I have vpn
but not that one so I can't watch most channels :(

**Comment by simakuutio** [comment]
> the channels I would like to watch are blocked saying I need vpn I have vpn but not that one so I can't watch most channels :(

Can you watch those same channel from website without VPN?

**Comment by suzie59** [comment]
No, I can't.  if you mean temp tv ustvgo will not let me i disconnected vpn
and still would not work :(

**Comment by Spintrin99** [comment]
Usatvgo and Allsprk stopped working. Can you please take a look?

**Comment by Tempest0580** [comment]
> Usatvgo and Allsprk stopped working. Can you please take a look?

Pushing a update soon to fix UstvGo soon, Allsprk seems to be working fine on my end.

**Comment by Spintrin99** [comment]
Usatvgo stopped working. Can you please check on your end. Thank you

**Comment by simakuutio** [comment]
I was kinda expecting this to happen. ustv247.tv earlier this week got all their channels blocked behind NordVPN and now us ustvgo have the same... I contacted my friend living in Florida and he can't access those channels either even he have a native IP from US so from now on, all those channels can be watched only if you're using NordVPN (no other VPN works, tested with various). It would be nice to hear, if anyone have NordVPN, does those even work with it?

**Comment by derrgoo** [comment]
Actually, I use ExpressVPN and I can watch.

**Comment by Tempest0580** [comment]
Pushed a update to fix UstvGo, Code Changed. Thanks

**Comment by deputydoobie** [comment]
Still down with surfshark, should I just get expressvpn?
Foreign channels are workingish

**Comment by Tempest0580** [comment]
For UstvGo and the VPN channels look here https://github.com/Tempest0580/tempest/issues/42.
Some people have said if you change the location on the VPN they are able to get all the channels work. I don't use a VPN and don't watch live TV so i can really help with that. Thanks

**Comment by derrgoo** [comment]
> For UstvGo and the VPN channels look here [Tempest0580/tempest#42](https://github.com/Tempest0580/tempest/issues/42).
> Some people have said if you change the location on the VPN they are able to get all the channels work. I don't use a VPN and don't watch live TV so i can really help with that. Thanks

Yes- the fox worked! Thank you. Just have to switch on your vpn and new version opens and plays.

**Comment by Spintrin99** [comment]
Usatvgo issue still. I click on a channel and it acts like its gonna work, but then does nothing. Thank you in advance for your assistance.

**Comment by derrgoo** [comment]
> Usatvgo issue still. I click on a channel and it acts like its gonna work, but then does nothing. Thank you in advance for your assistance.

Use a vpn. It works perfectly.

**Comment by Spintrin99** [comment]
> > Usatvgo issue still. I click on a channel and it acts like its gonna work, but then does nothing. Thank you in advance for your assistance.
> 
> Use a vpn. It works perfectly.

I am using IPVanish. Should I change to another?

**Comment by derrgoo** [comment]
> > > Usatvgo issue still. I click on a channel and it acts like its gonna work, but then does nothing. Thank you in advance for your assistance.
> > 
> > 
> > Use a vpn. It works perfectly.
> 
> I am using IPVanish. Should I change to another?

Try to change to Singapore or another country. That works for me. USA did not work.

**Comment by suzie59** [comment]
I just tried oxygen and it worked?>

**Comment by suzie59** [comment]
thank you @tempest0580

**Comment by derrgoo** [comment]
Any chance this will be upgraded to kodi 19 matrix? Thank you!

**Comment by suzie59** [comment]
Im using 19 on mine and tempest works

**Comment by derrgoo** [comment]
> Im using 19 on mine and tempest works
> […](#)

When I install it shows tempest installed but can’t get to video add ons to install. Kodi 19rc

**Comment by suzie59** [comment]
i didnt know you could log into ustreamix what is that

**Comment by deputydoobie** [comment]
No sound on tnt, is that normal?

**Comment by derrgoo** [comment]
> No sound on tnt, is that normal?

It’s normal. Just provider having an issue. Nothing dev can do.

**Comment by Spintrin99** [comment]
Usatvgo issue: message states the channels are VPN locked or the code has changed.
Can you please look into this?
Thank you in advance.

**Comment by suzie59** [comment]
same here :(

**Comment by Dimension02000** [comment]
Looks like the issue of  the VPN locked or the code has changed is still taking place.

**Comment by suzie59** [comment]
I changed over to the crew
